tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -3767,4 +3767,46 @@ public function createPriceSetWithPage($entity = NULL, $params = array()) {
     $this->_ids['membership_type'] = $membershipTypeID;
   }
 
+  public function setupMultipleMembershipPriceset($number = 1) {
```

Comment:
Just a note that I think we are going to have to move to have comment blocks for functions in the test suite at some point - not a must-fix-now - but something to try to do next time.

If we run phpcs over the code & want to raise the bar on our standards I don't think we can have higher standards on some directories than others - I think Tim tried.